aspnetcore/blazor/tooling.md

```diff
@@ -448,13 +431,278 @@ The following example from the [tutorial for using SignalR with a Blazor WebAsse
+
+Select **Yes** if you trust the ASP.NET Core SSL certificate.
+
+The following dialog is displayed:
+
+![Security warning dialog](~/blazor/tooling/_static/install-certificate.png)
+
+Select **Yes** to acknowledge the risk and install the certificate.
+
+For information on trusting the development certificate for the Firefox browser, see <xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error>.
+
+Visual Studio:
+
+* Compiles and runs the app.
+* Launches the default browser at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error).
+
+In VS Code, press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app without debugging.
+
+At the **Select debugger** prompt in the **Command Palette** at the top of the VS Code UI, select **C#**. At the next prompt, select the HTTPS profile (`C#: BlazorWebAppMovies [https]`).
+
+The default browser is launched at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+:::zone-end
+
+:::zone pivot="cli"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error) section of that article.
+
+In a command shell opened to the project's root folder, execute the [`dotnet run`](/dotnet/core/tools/dotnet-run) command to compile and start the app:
+
+```dotnetcli
+dotnet run
+```
+
+The default browser is launched at `http://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
```

Comment:
Hmm, I don't think the browser will auto launch if you do `dotnet run`. I think that's only a feature of `dotnet watch`.

Reply:
Yes, "launched" is incorrect. It should probably say, "Open a browser at the localhost URL shown in the console." Then, it can mention that the port is randomly assigned at app generation.

Reply:
***Done!*** 👍 ... Will be on the next commit.